test/pylib/scylla_cluster.py

```diff
@@ -52,6 +52,132 @@
 
 io_executor = concurrent.futures.ThreadPoolExecutor(max_workers=8)
 
+certificate_file = """
+-----BEGIN CERTIFICATE-----
```

Comment:
Please move these certificates to the files and read them here instead of storing full certificate. I suppose you can put them in the directory test/pylib/resources